trunk/SUAVE/Components/Energy/Converters/Rotor.py

```diff
@@ -186,7 +190,10 @@ def spin(self,conditions):
         V_thrust        = orientation_product(T_body2thrust,V_body) 
     
         # Now just use the aligned velocity
-        V        = V_thrust[:,0,None] + V0 
+        if np.any(theta  > np.pi/3):
```

Comment:
I don't think this check to add inflow velocity makes sense based off of thrust angle. Also the angle itself seems arbitrary. How about you check if the axial flow is some percentage of the velocity magnitude, that would mean that the rotor is in true edgewise flight.

Reply:
true good point. I have this flag in another branch, will use this flag instead. Thanks